inst/rmarkdown/templates/nutrition/skeleton/skeleton.Rmd

```diff
@@ -85,6 +90,21 @@ plot_age_pyramid(filter(survey_data, !is.na(age_group)))
 ```
 
 
+
+```{r}
+survey_data %>%
+  mutate(total = 1) %>% 
+  select(sex, age_group, age_24months, total) %>% 
+  filter(!is.na(age_group)) %>% 
+  tidyr::gather(class, flagged) %>% 
+  count(class, flagged) %>% 
+  group_by(class) %>% 
+  mutate(prop = prop.table(n)) %>% 
+  knitr::kable()
+
+```
+
+
 ## Anthropometric results (based on WHO standards 2006)
 
 ```{r}
```

Comment:
can you move libary ggplot to the top so there is oversight on what is used - see cholera template